scripts/docker-run.sh

```diff
@@ -64,6 +64,7 @@ $DOCKER_RUN --rm \
 	--env DELPHIX_SIGNATURE_VERSIONS \
 	--env DELPHIX_UPGRADE_MINIMUM_VERSION \
 	--env DELPHIX_UPGRADE_MINIMUM_REBOOT_OPTIONAL_VERSION \
+	--env UPSTREAM_PRODUCT_BRANCH \
```

Comment:
Are you still planning to use this variable?

Based on your recent changes to `run-live-build.sh`, this isn't being used.